spec/models/hackney/repair_request_spec.rb

```diff
@@ -104,7 +104,7 @@
       ],
       priority: "G",
       property_reference: "00000018",
-      description: "it's broken fix it",
```

Comment:
I feel you should separate this into another test. This is checking whether the method creates a repair request, you should have a separate one about the `squish` behaviour. Otherwise, it's not clear that this is intended.